### PR TITLE
When deleting a collections, records are deleted but tombstone aren't

### PR DESCRIPTION
**Pull request by Natim**
https://github.com/mozilla-services/kinto/blob/master/kinto/views/collections.py#L31-L35

```
echo '{"data": {"title": "toto"}}' | http POST http://localhost:8888/v1/buckets/default/collections/tasks/records --auth user:pass

http DELETE http://localhost:8888/v1/buckets/default/collections/tasks/records/667c36ba-5a27-45bc-b9a4-94f3d4d46bf4 --auth user:pass

http GET http://localhost:8888/v1/buckets/default/collections/tasks/records?_since=1436786412344 --auth user:pass

http DELETE http://localhost:8888/v1/buckets/default/collections/tasks --auth user:pass

echo '{"data": {}}' | http PUT http://localhost:8888/v1/buckets/default/collections/tasks --auth user:pass

http GET http://localhost:8888/v1/buckets/default/collections/tasks/records?_since=1436786412344 --auth user:pass
HTTP/1.1 200 OK
Access-Control-Expose-Headers: Backoff, Retry-After, Alert, Next-Page, Total-Records, Last-Modified, ETag
Content-Length: 101
Content-Type: application/json; charset=UTF-8
Date: Mon, 13 Jul 2015 11:20:52 GMT
Etag: "1436786412345"
Last-Modified: Mon, 13 Jul 2015 11:20:12 GMT
Server: waitress
Total-Records: 0

{
    "data": [
        {
            "deleted": true, 
            "id": "667c36ba-5a27-45bc-b9a4-94f3d4d46bf4", 
            "last_modified": 1436786412345
        }
    ]
}
```

### CODE REVIEW

**Comment by Natim** [comment]
Needs mozilla-services/cliquet#400 to be merged first.

**Comment by leplatrem** [comment]
LGTM !

Just wondering : why sometimes with_deleted and sometimes not ?

File: kinto/views/buckets.py [hunk 1]
```diff
@@ -118,16 +118,24 @@ def delete(self):
         storage = self.collection.storage
         parent_id = '/buckets/%s' % self.record_id
         storage.delete_all(collection_id='group', parent_id=parent_id)
```
Comment (Natim) [inline kinto/views/buckets.py]: Missing a ``with_deleted=False``

File: kinto/views/collections.py [hunk 2]
```diff
@@ -33,5 +33,6 @@ def delete(self):
         parent_id = '%s/collections/%s' % (self.collection.parent_id,
                                            self.record_id)
         storage.delete_all(collection_id='record', parent_id=parent_id)
```
Comment (Natim) [inline kinto/views/collections.py]: missing ``with_deleted=False``

**Pull request merged**